libmcount/mcount.c

```diff
@@ -543,6 +544,9 @@ int mcount_entry(unsigned long *parent_loc, unsigned long child,
 	/* fixup the parent_loc in an arch-dependant way (if needed) */
 	parent_loc = mcount_arch_parent_location(&symtabs, parent_loc, child);
 
+	/* python hooking for function entry */
+	python_mcount_entry(child, *parent_loc);
```

Comment:
Oh, and I want to call the function only if script file is given in order to reduce the overhead.  Ditto for the other callsites below..